src/ansible.rs

```diff
@@ -140,9 +140,6 @@ impl AnsibleRunnerInterface for AnsibleRunner {
             .map(|(name, vars)| (name, vars.ansible_host))
             .collect();
 
-        if inventory.is_empty() {
```

Comment:
Why would you not want an error for empty inventory?

Reply:
During `list_inventory`, we don't have info about whether the `build` machine was created/not during `deploy`. 
Thus, it is better to check for `empty invetory` after the `inventory_list() fn` is called instead of propagating error from it. 

The other option is to handle the error case.
Or we should not be writing `.{name}_build_inventory_digital_ocean.yml` file if the build vm is not setup. I haven't tried going that route as it was not obvious to me.

Reply:
OK no problem.